zone.dos.Browser.yaml

```diff
@@ -0,0 +1,66 @@
+id: zone.dos.Browser
```

Comment:
I don't see anything that connects dos.zone with https://github.com/js-dos/dos.zone.browser.pc Do you control the webpage?

The github org has a different unverified website js-dos.com

Reply:
Yeah, I own both dos.zone and js-dos.com. I can verify.

Reply:
can you put something up linking this project or submission in dos.zone webpage?

Reply:
sure, 
https://dos.zone/flathub-verify.txt

if you need something else, please ask

Reply:
thanks.